apps/system-e2e/src/integration/web.spec.ts

```diff
@@ -28,4 +28,32 @@ describe('web', () => {
           .should('equal', '/'),
       )
   })
+
+  it('should have search results for common words', function () {
+    cy.visit('/')
+      .get('[data-testid="search-box"]')
+      .click()
+      .type('covid{enter}')
```

Comment:
Covid articles may disappear in the future

Reply:
Fixed 👍🏻